docs/_source/guides/features/listeners.ipynb

```diff
@@ -0,0 +1,244 @@
+{
+ "cells": [
+  {
+   "attachments": {},
+   "cell_type": "markdown",
+   "id": "025f037c-bf3b-48f9-b347-e325737c07a0",
+   "metadata": {},
+   "source": [
+    "# Listeners\n",
+    "\n",
+    "This guide gives you a brief introduction to Argilla Listeners. Argilla Listeners enable you to build fine-grained complex workflows as background processes, like a low-key alternative to job scheduling directly integrated with Argilla. \n",
+    "\n",
+    "The main goal facilitate the user to define and customize their Argilla experience, and these Listeners are the back-bone of our [Argilla Plugins](https://github.com/argilla-io/argilla-plugins). Note that the tutorial about [active learning with small-text](../../tutorials/libraries/smalltext.md) is a great example of how powerful listeners can be. Alternatively, you can check the [Python Client](../../reference/python/python_listeners.rst) to get acquinted.\n",
+    "\n",
+    ".. nbinfo::\n",
```

Comment:
Use this instead so it's shown with the doc css styles:

<div class="alert alert-info">
</div>

Reply:
I think it is a cache, what should I use?

Reply:
ah sorry! I added the html snippet directly and it seems to remove it, now:

```html
<div class="alert alert-info">
</div>
```